core/math/expression.cpp

```diff
@@ -452,16 +452,16 @@ Error Expression::_get_token(Token &r_token) {
 					} else if (id == "false") {
 						r_token.type = TK_CONSTANT;
 						r_token.value = false;
-					} else if (id == "PI") {
+					} else if (id == "PI" || id == "pi") {
```

Comment:
What about `id.nocasecmp_to("pi") == 0` ? Would allow Pi, TaU or iNf as well, not sure if it's desired.

Reply:
Good idea! Actually, I think an even better option would be to support this for all constants / operators, so Python users will feel at home writing `True` and such, SQL users will feel at home writing `AND`/`OR`/`NOT`, and users won't have to remember the capitalization.